disneyland/server.go

```diff
@@ -18,45 +18,42 @@ func detailedInternalError(err error) error {
 		fmt.Sprintf("Error processing job: %v", err),
 	)
 }
-func validateAccess(project string, userProject string, kind string, userKind string) (string, string, error) {
-	if kind == "" {
-		if userKind == "ANY" {
-			return "", "", grpc.Errorf(codes.DataLoss, "ListJobsKindRequest.Kind not specified")
-		}
-		kind = userKind
+func (s *Server) Init() {}
```

Comment:
there should be empty lines before and after this method. Also Init() seem to be empty, maybe you should remove it.